scripts/mkcloudhost/cloudfunc

```diff
@@ -10,6 +10,12 @@ function cloudadminnet()
     echo -n 192.168.$((255-2*n))
 }
 
+function cloudironicnet()
+{
+    n=$1
+    echo -n 192.168.$((4*n))
```

Comment:
This was proposed by @bmwiedemann when I first configured this on one of our shared hosts. For high `n` it can overlap the other networks but I don't think we use all 32 slots in many places. If someone has a better idea I don't see a problem changing it.